### PR TITLE
Add nfts_by_issuer data type

### PR DESCRIPTION
**Pull request by Kassaking7**
## High Level Overview of Change
Title

### Context of Change
Following https://ripplelabs.atlassian.net/jira/software/projects/DEFI/boards/760?assignee=712020%3Aff4deb34-7fc1-428e-a2ee-354e1ddba309&selectedIssue=DEFI-128 & https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer/

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### Did you update CHANGELOG.md?

- [x] Yes
- [ ] No, this change does not impact library users

### CODE REVIEW

**Comment by Kassaking7** [comment]
Seems not require test cases since the lib currently doesn't support integration test for Clio

File: xrpl/models/requests/nfts_by_issuer.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from typing import Any, Optional, Union
+
+from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.required import REQUIRED
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class NFTsByIssuer(Request):
```
Comment (shawnxie999) [inline xrpl/models/requests/nfts_by_issuer.py:15]: can inherit LookupByLedgerRequest, and omit ledger_hash and ledger_index

**Comment by shawnxie999** [comment]
You are missing some changes. You can take a look at how the nft_history API is done. Do global search to find `NFT_HISTORY` and `NFTHistory`, and see where you are missing in some of the places.

File: xrpl/models/requests/nfts_by_issuer.py [hunk 2]
```diff
@@ -5,28 +5,22 @@
...
     The `nfts_by_issuer` method retrieves all of the NFTokens
     issued by an account
     """
 
     method: RequestMethod = field(default=RequestMethod.NFTS_BY_ISSUER, init=False)
     issuer: str = REQUIRED  # type: ignore
-    """
-    This field is required.
-    :meta hide-value:
-    """
```
Comment (shawnxie999) [inline xrpl/models/requests/nfts_by_issuer.py]: this comment block is still needed as it describes the issuer param. You also should make it more descriptive, something like "The unique identifier for an account"

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Support for the Price Oracles amendment (XLS-47).
+- Add nfts_by_issuer data type
```
Comment (shawnxie999) [inline CHANGELOG.md:13]: should be `nfts_by_issuer`

**Review by shawnxie999 — APPROVED** [review]
looks good :)

**Review by pdp2121 — APPROVED** [review]
LGTM!

**Pull request merged**